e2e/tests/container_list.go

```diff
@@ -226,7 +226,7 @@ func ContainerList(opt *option.Option) {
 			body, err := io.ReadAll(res.Body)
 			Expect(err).Should(BeNil())
 			defer res.Body.Close()
-			errorMsg := fmt.Sprintf("invalid query parameter \\\"filters\\\": %s", fmt.Errorf("invalid character 'i' looking for beginning of value"))
+			errorMsg := fmt.Sprintf("invalid query parameter \\\"filters\\\": %s", fmt.Errorf("error parsing filters: invalid character 'i' looking for beginning of value"))
```

Comment:
Can you also add a e2e tests with the new filter format?

Reply:
added, https://github.com/runfinch/finch-daemon/blob/a3b76dd5428f552f73b69fd5ae93a3738eb77120/e2e/tests/container_list.go#L232